pipeline/common_stages.groovy

```diff
@@ -319,8 +319,14 @@ def deploy_mig_controller_on_both(
   }
 
   return {
-    stage('Deploy mig-controller on both clusters') {
-      steps_finished << 'Deploy mig-controller on both clusters'
+    stage('Build mig-controller image and deploy on both clusters') {
+      steps_finished << 'Build mig-controller image and deploy on both clusters'
+      // Create mig-controller docker image
+      withEnv(["IMG=quay.io/ocpmigrate/mig-controller:${MIG_CONTROLLER_BRANCH}"]) {
+        dir('mig-controller') {
```

Comment:
I don't think the mig-controller repo is actually cloned at this stage, we clone it the first time the deployment playbooks run...